adaptive/learner/learner1D.py

```diff
@@ -580,22 +580,33 @@ def _loss(self, mapping, ival):
         loss = mapping[ival]
         return finite_loss(ival, loss, self._scale[0])
 
-    def plot(self):
+    def plot(self, *, scatter_or_line=None):
         """Returns a plot of the evaluated data.
 
         Returns
         -------
         plot : `holoviews.element.Scatter` (if vdim=1)\
                else `holoviews.element.Path`
             Plot of the evaluated data.
+        scatter_or_line : str, optional
+            Plot as a scatter plot ("scatter") or a line plot ("line").
+            By default a line plot will be chosen if the data consists of
+            vectors, otherwise it is a scatter plot.
         """
         hv = ensure_holoviews()
 
         xs, ys = zip(*sorted(self.data.items())) if self.data else ([], [])
         if self.vdim == 1:
-            p = hv.Path([]) * hv.Scatter((xs, ys))
+            if scatter_or_line is None or scatter_or_line == "scatter":
```

Comment:
once `scatter_or_line` has a default value of `scatter` you can change this test and the one below to be `if scatter_or_line == "scatter`

Reply:
Simplified it a bit more.